containers/haskell/.devcontainer/Dockerfile

```diff
@@ -3,9 +3,9 @@ FROM debian:bullseye-slim
 ENV LANG C.UTF-8
 
 # [Option] Install zsh
-ARG INSTALL_ZSH="true"
+ARG INSTALL_ZSH="${templateOption:installZsh}"
```

Comment:
The Stack Overflow question [unable to build VSC devcontainer for Haskell on OS X Ventura](https://stackoverflow.com/q/74929464/7509065) appears to be related to this change.

Reply:
Thanks! Replied my thoughts on the thread.